src/java/org/apache/nutch/indexer/IndexWriters.java

```diff
@@ -211,7 +211,7 @@ private Collection<String> getIndexWriters(NutchDocument doc) {
   public void open(Configuration conf, String name) throws IOException {
     for (Map.Entry<String, IndexWriterWrapper> entry : this.indexWriters
         .entrySet()) {
-      entry.getValue().getIndexWriter().open(conf, name);
+      entry.getValue().getIndexWriter().open(new IndexWriterParams(new HashMap<>()));
```

Comment:
Are these the correct method parameters if we were to remove the deprecated `open(Configuration conf, String name)` method?